src/tests/unit/auto/auto_ctput_test.cpp

```diff
@@ -0,0 +1,255 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <common_test_utils/test_constants.hpp>
+#include <ie_core.hpp>
+#include <ie_metric_helpers.hpp>
+#include <multi-device/multi_device_config.hpp>
+#include <ngraph_functions/subgraph_builders.hpp>
+#include <openvino/runtime/core.hpp>
+
+#include "cpp/ie_plugin.hpp"
+#include "mock_common.hpp"
+#include "plugin/mock_load_network_properties.hpp"
+#include "unit_test_utils/mocks/cpp_interfaces/impl/mock_inference_plugin_internal.hpp"
+#include "unit_test_utils/mocks/cpp_interfaces/interface/mock_icore.hpp"
+#include "unit_test_utils/mocks/cpp_interfaces/interface/mock_iexecutable_network_internal.hpp"
+#include "unit_test_utils/mocks/cpp_interfaces/interface/mock_iinference_plugin.hpp"
+#include "unit_test_utils/mocks/cpp_interfaces/interface/mock_ivariable_state_internal.hpp"
+#include "unit_test_utils/mocks/mock_iinfer_request.hpp"
```

Comment:
Do we really need all these headers? Or is it a copy/past? I don't see that we need for example ie_core.hpp
Also please use "" instead of <> for openvino headers

Reply:
updated

Reply:
Remove the redundant header file of auto_ctput_test.cpp